danielMunteanu/src/main/java/Generics/newspapers/ZiarulDeGarda.java

```diff
@@ -0,0 +1,27 @@
+package Generics.newspapers;
+
+import Generics.LibraryItemType;
+import Generics.Newspaper;
+
+public class ZiarulDeGarda implements Newspaper {
+    @Override
+    public boolean political() {
```

Comment:
would be nice to have all the gender types stored as an ENUM, then you can define this method as isTypeOf(Gender dender){}

Reply:
done